### PR TITLE
PD-JS1-W4

### PR DESCRIPTION
**Pull request by esma-g**
## What does this change?

Module: JS1
Week(s): 4

## Description

- Deleted learning objectives
- Removed notes for the facilitator
- PD session content and coursework added
- Coursework formatted with number
- Session and exercise objectives added

## Who needs to know about this?

@kfklein15 @ElizabethlawalCYF @SallyMcGrath @CodeYourFuture/syllabus

### CODE REVIEW

File: docs/js-core-1/week-4/homework.md [hunk 1]
```diff
@@ -76,3 +76,9 @@ Have some extra time before our next class? Fill it with these harder challenges
 **Alert:** Some of these challenges might include concepts that we haven't taught you yet. You should us Google to try to learn how to solve them.
 
 https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week3
+
+## Writing a meeting request email (PD - 30 min)
+
+In the following scenario imagine you are a recruit for a company and your team leader asks you to write an email requesting a meeting with the Head of Customer Services (HCS). Remember to include the subject line and use appropriate formatting.
+
+"HCS has complained about the customer services application to the software team leader. He has found some frustrating problems with the user interface. The team leader told you to go and see the HCS, find out what is wrong and then come back with improvements. The HCS is very busy. You should email requesting a meeting and outline clearly what you needs the meeting for."
```
Comment (kfklein15) [inline docs/js-core-1/week-4/homework.md]: "clearly what you need the meeting for" - no S on the need.

File: docs/js-core-1/week-4/lesson.md [hunk 2]
```diff
@@ -559,6 +560,22 @@ If you're using the above example, you should expect to return an array with 2x
...
+
+[Adapted from Tectonica workshop](https://github.com/Techtonica/curriculum/blob/main/onboarding/asking-good-questions.md)
+
+[Asking Good Questions slides](https://docs.google.com/presentation/d/19QbXNYHj8v-xSg9PT3MDORxpEiNh2UGt1eIuYYgvCVY/edit#slide=id.gaa71b86f5e_0_0)
+
+Notes for the facilitator: Remember to adapt/update the content in line with your cohort’s needs and arrange timings as needed.
+
+:::note Exercise (15 min)
+
+In small groups. Pick a programming concept that you have difficulty with or are trying to learn. Use StackOverflow to see if you can get some good information about it. Role play for asking questions with a partner, use the the template sample script:
```
Comment (kfklein15) [inline docs/js-core-1/week-4/lesson.md]: I don't think we need the full stop on this phrase
"In small groups, pick a programming concept that you have difficulty with or are trying to learn."

**Review by kfklein15 — CHANGES_REQUESTED** [review]
Some small changes.

File: docs/js-core-1/week-4/homework.md (hunk 1 above)
Comment (esma-g) [reply docs/js-core-1/week-4/homework.md]: Done

File: docs/js-core-1/week-4/lesson.md (hunk 2 above)
Comment (esma-g) [reply docs/js-core-1/week-4/lesson.md]: Done

**Review by kfklein15 — DISMISSED** [review]
LGTM

**Review by kfklein15 — DISMISSED** [review]
changes done, so all ok

**Comment by esma-g** [comment]
Deleting PD learning objectives and not adding any new ones until they are re-created by the PD syllabus review.

**Review by kfklein15 — APPROVED** [review]
LGTM

**Review by illicitonion — APPROVED** [review]

**Pull request merged**